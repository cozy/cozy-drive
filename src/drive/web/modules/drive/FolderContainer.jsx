import React from 'react'

import Container from './Container'

const FolderContainer = props => (
  <Container
    isTrashContext={false}
    canSort
    canDrop
    canUpload
    canCreateFolder
    canMove
    {...props}
  />
)

export default FolderContainer
