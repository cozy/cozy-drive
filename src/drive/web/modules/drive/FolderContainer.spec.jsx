import { shallow } from 'enzyme'
import React from 'react'

import FolderContainer from './FolderContainer'

describe('FolderContainer', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(
      <FolderContainer
        additionalProp="foobar"
        canSort={true}
        canDrop={true}
        canUpload={true}
        canCreateFolder={true}
        canMove={true}
      />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
