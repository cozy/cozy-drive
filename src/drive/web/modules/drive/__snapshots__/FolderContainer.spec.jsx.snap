// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FolderContainer should match the snapshot 1`] = `
<FolderViewWithSharingContext
  additionalProp="foobar"
  canCreateFolder={true}
  canDrop={true}
  canMove={true}
  canSort={true}
  canUpload={true}
  isTrashContext={false}
/>
`;
